src/components/Common/Tabs/Tabs.js

```diff
@@ -31,23 +32,30 @@ export const Tabs = ({
     if (selectedTab !== activeTab) setSelectedTab(activeTab);
   }, [selectedTab, activeTab]);
 
-  return (
-    <TabsContext.Provider value={{
+  const contextValue = useMemo(() => {
+    return {
       switchTab,
       selectedTab,
+      allowedActions,
       lastTab: children.length === 1,
-    }}>
+    };
+  }, [switchTab, selectedTab, allowedActions, children.length]);
+
+  return (
+    <TabsContext.Provider value={contextValue}>
       <Block name="tabs">
         <Elem name="list">
           {children}
 
-          <Elem
-            tag={Button}
-            name="add"
-            type="text"
-            onClick={onAdd}
-            icon={addIcon}
-          />
+          {allowedActions.add !== false && (
```

Comment:
Is this action implicitly true, explicitly false?

Reply:
`undefined` will be an edge case as the default value for `allowedActions` is "all true":

https://github.com/heartexlabs/dm2/pull/132/files#diff-0b0b6f242eeb3144398458132815122cea9e185111bf094fbc4db7133d80fe14R130-R135